src/database/Database.cpp

```diff
@@ -51,7 +51,7 @@ using namespace std;
 
 bool Database::gDriversRegistered = false;
 
-static unsigned long const SCHEMA_VERSION = 8;
+static unsigned long const SCHEMA_VERSION = 9;
```

Comment:
no need to bump the version: I imagine that we're going to merge this into the upcoming release